docs/docsite/rst/dev_guide/developing_modules_documenting.rst

```diff
@@ -297,8 +297,10 @@ The parameters for these formatting functions can use quoting with backslashes:
 Rules for using ``O()`` and ``RV()`` are very strict. You must follow syntax rules so that documentation renderers can create hyperlinks for the options resp. return values.
 
 The allowed syntaxes are as follows:
-- To reference an option for the current plugin/module, use ``O(option)`` and ``O(option=name)``.
-- To reference an option for *another* plugin/module, use ``O(plugin.fqcn.name#type:option)`` and ``O(plugin.fqcn.name#type:option=name)``. For modules, use ``type=module``. The FQCN and plugin type can be ignored by the documentation renderer, turned into a link to that plugin, or even directly to that plugin's option.
+- To reference an option for the current plugin/module, or the current role's entrypoint (inside a role's entrypoint documentation), use ``O(option)`` and ``O(option=name)``.
```

Comment:
I know it increases the word count but I generally discourage the possessive form because it can be confused with verb contractions and is a little anthropomorphic.

s / or the current role's entrypoint / or the entrypoint of the current role

I'd also maybe change the other instances of possessive in the following lines. 

s / a role's documentation / role documentation 
s / that entrypoint's option / an entrypoint option

Reply:
Thanks! I tried to improve that in 055327c5b66a586d47bcb3a6613d41fcd8d2efb0 using your suggestions, can you check the result?